aepp/configs.py

```diff
@@ -124,6 +128,7 @@ def configure(
         private_key : REQUIRED : If you do not use a file but pass a variable directly.
         sandbox : OPTIONAL : If not provided, default to prod
         connectInstance : OPTIONAL : If you want to return an instance of the ConnectObject class
+        environment : OPTIONAL : If not provided, default to prod
     """
     if not org_id:
         raise ValueError("`org_id` must be specified in the configuration.")
```

Comment:
what do. you think about creating annotations that users can use, for example an annotation for the connection that specifies env as a param as well as some of these other parameters

Reply:
Unclear how an end-user would use annotations, compare to for example `aepp.configure(environment="prod")` - do you have an example of what you have in mind?